mirrord/config/src/lib.rs

```diff
@@ -116,6 +116,9 @@ pub struct LayerConfig {
     /// Controls mirrord features, see [`feature::FeatureFileConfig`].
     #[config(nested)]
     pub feature: FeatureConfig,
+
+    #[config(env = "MIRRORD_OPERATOR")]
+    pub operator: Option<String>,
```

Comment:
I think we should check on start if operator exist and just use that instead of adding a flag

Reply:
Adding it in different PR this one already too big